Main.java

```diff
@@ -0,0 +1,401 @@
+package org.example;
```

Comment:
Kind of. The folder containing the Main.java file is named 'org.example', so that line was necessary because I forgot to change it to something more meaningful.
I changed it to 'org.eisop'. Let me know if you want it to be something else.

Reply:
Never mind. Changed to io.github.eisop.website.